graph/config/cytoscape/cytoscape.go

```diff
@@ -112,6 +112,7 @@ type NodeData struct {
 	HasTrafficShifting    bool                `json:"hasTrafficShifting,omitempty"`    // true (vs has traffic shifting) | false
 	HasVS                 *VSInfo             `json:"hasVS,omitempty"`                 // it can be empty if there is a VS without hostnames
 	HasWorkloadEntry      []graph.WEInfo      `json:"hasWorkloadEntry,omitempty"`      // static workload entry information | empty if there are no workload entries
+	IsAutogenerated       bool                `json:"isAutogenerated,omitempty"`       // true (object is auto-generated from K8s API Gateway) | false
```

Comment:
```suggestion
	IsK8sGatewayAPI       bool                `json:"isK8sGatewayAPI,omitempty"`       // true (object is auto-generated from K8s API Gateway) | false
```

I think `IsAutogenerated` is too much of an implementation detail.  Currently I guess Istio must be autogenerating K8s GW API definitions into the existing Virtual Service definition and assigning a special name.  I'm not sure this will always be true and I think it would just be more clear with a name change.

A name change would require a little bit of re-alphabetizing...